callback/bpmn.go

```diff
@@ -215,17 +219,9 @@ func updatePolicy(state *bpmn.State) error {
 		return err
 	}
 
-	// Update agency if present
-	err = models.UpdateAgencyPortfolio(policy, origin)
-	if err != nil && err.Error() != "agency not set" {
-		log.Printf("[updatePolicy] ERROR updateAgencyPortfolio %s", err.Error())
-		return err
-	}
-
-	// Update agent if present
-	err = models.UpdateAgentPortfolio(policy, origin)
-	if err != nil && err.Error() != "agent not set" {
-		log.Printf("[updatePolicy] ERROR UpdateAgentPortfolio %s", err.Error())
+	err = network.UpdateNetworkNodePortfolio(origin, policy, networkNode)
+	if err != nil {
+		log.Printf("[updatePolicy] error updating %s portfolio %s", networkNode.Type, err.Error())
```

Comment:
how does it behave with e-commerce where we do not have a networkNode. Previously we would check for "agency/agent not set" error. We might need to call this update function only if we do have a node, otherwise change it to have an early non-error escape for a missing node.